docs/notebooks/lightcurve_source_demo.ipynb

```diff
@@ -112,7 +114,14 @@
    "id": "188eba77",
    "metadata": {},
    "source": [
-    "We then create the model from the dictionary of light curve information "
+    "We then create the model from the dictionary of light curve information. \n",
+    "\n",
+    "In addition to the light curve data itself, the model also needs to know:\n",
+    "\n",
+    "  * `t0` - The query time (in MJD) that corresponds to the model’s reference epoch.\n",
+    "  * `lc_data_t0` - The time stamp of the input data (`lightcurves`) that correspond to the model’s `t0`.\n",
```

Comment:
```suggestion
    "  * `t0` - The survey date (e.g. in MJD) that corresponds to the model’s reference epoch.\n",
    "  * `lc_data_t0` - The time stamp of the input data (`lightcurves`) specifying the model’s reference epoch. If the survey made an observation at time `t0`, it would measure the band flux corresponding to the `lc_data_t0` timestamp.\n",
```